saleor/payment/gateways/adyen/webhooks.py

```diff
@@ -226,6 +226,22 @@ def handle_authorization(notification: Dict[str, Any], gateway_config: GatewayCo
     kind = TransactionKind.AUTH
     if adyen_auto_capture:
         kind = TransactionKind.CAPTURE
+
+    amount = notification.get("amount", {})
+    notification_payment_amount = price_from_minor_unit(
+        amount.get("value"), amount.get("currency")
```

Comment:
if amount.get("value") returns `None` (e.g when amount will be set to `{}`),  `price_from_minor_unit` will raise TypeError. Are we sure that amount will always have proper data inside?